Python/optimizer_analysis.c

```diff
@@ -392,9 +392,12 @@ optimize_uops(
         ctx->frame->stack_pointer = stack_pointer;
         assert(STACK_LEVEL() >= 0);
     }
-
+    if (this_instr != trace + trace_len) {
+        assert (this_instr < trace + trace_len && this_instr > trace);
+        trace_len = this_instr - trace + 1;
```

Comment:
Needs a cast here, warnings aplenty in GH
```suggestion
        trace_len = (int)(this_instr - trace) + 1;
```